lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -2,42 +2,41 @@
 """
 Tests of verify_student views.
 """
+
+import ddt
 import json
+import pytz
```

Comment:
`pytz` isn't part of the standard library either; please move it to the second block of imports.